IRS/irs_scraper.py

```diff
@@ -3,12 +3,27 @@
 from pymongo import MongoClient, TEXT
 import random
 import os
+import sys
 from tqdm import tqdm
 import re
 from collections import OrderedDict
 import requests
 from datetime import datetime, date
 
+if __package__:  # if script is being run as a module
+    from ..shelterapputils.utils import check_similarity, refresh_ngrams
```

Comment:
this piece of code is also common in both scrapers, is it possible to have single copy of it.?

Reply:
This is necessary to import files that aren't in the same directory of the scraper, so we need to have it in both places